framework/db/config_manager.py

```diff
@@ -29,6 +30,18 @@ def IsConvertable(self, value, conv):
 
     def LoadConfigDBFromFile(self, file_path):
         # TODO: Implementy user override mechanism
+        path = os.path.join(os.path.expanduser("~"), os.path.join('.owtf', 'configuration'))
```

Comment:
Do not take absolute path in the codebase. Use `framework_config.cfg` to take appropriate value.
What is your view @delta24